### DISCUSSION TITLE
A map that does not rotate

### DISCUSSION THREAD

**Discussion by jackfioru92** [fleaflet/flutter_map#1791] [Q&A]
In Flutter Map version 5.0, to prevent map rotation, I used 'nonRotatedChildren:' instead of 'children' like this:

```dart
FlutterMap(
  mapController: _mapController,
  options: MapOptions(
  center: userPosition,
  zoom: userPosition != defaultLocation ? 17 : 5,
  minZoom: 5,
  maxZoom: 18,
  interactiveFlags: InteractiveFlag.all & ~InteractiveFlag.rotate,
  onTap: (tapPosition, point) => searchFocusNode.unfocus(),
),
nonRotatedChildren: <Widget>[
TileLayer()
```

 Now, I see that starting from version 6 of Flutter Map, this is no longer possible. How can I modify my code to achieve the same result?

**Comment by JaffaKetchup** [comment]
Hi @jackfioru92,
This way of doing this has been removed because it was actually not the correct way to prevent rotation. The line you have included above,
```dart
interactiveFlags: InteractiveFlag.all & ~InteractiveFlag.rotate,
```
,should have been enough.
To migrate this to v6, just move these flags as specified in the migration instructions: https://docs.fleaflet.dev/getting-started/migrating-to-v6#interactive-options-such-as-interactiveflags-have-been-moved-into-interactionoptions.
If this doesn't resolve the issue, please let us know!
